### PR TITLE
feat: add real-life oauth client ids

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

**Comment by prusnak** [comment]
Updated the PR with the new GOOGLE_CLIENT_ID

File: packages/suite/src/utils/suite/oauth.ts [hunk 1]
```diff
@@ -11,11 +12,8 @@ export const getOauthReceiverUrl = () => {
     if (!window.ipcRenderer) {
         return `${window.location.origin}${getPrefixedURL('/static/oauth/oauth_receiver.html')}`;
     }
-
-    // todo: upload oauth_receiver.html from suite to trezor.io (it contains code to handle ipcRenderer);
-    // return 'https://beta-wallet.trezor.io/oauth_receiver.html';
-    // todo: temporarily herokuapp is used.
-    return 'https://track-suite.herokuapp.com/oauth/';
+    // for desktop. but this solution is temporary, local http server will be used later to accept callback
+    return 'https://wallet.trezor.io/oauth_receiver.html';
```
Comment (keraf) [inline packages/suite/src/utils/suite/oauth.ts:16]: I would suggest prefixing temporary fix comments with a searchable term like `TEMP:` to make it easier to find and report.

File: packages/suite/src/services/metadata/DropboxProvider.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ class DropboxProvider implements AbstractMetadataProvider {
     }
 
     async connect() {
+        console.log(getOauthReceiverUrl());
```
Comment (prusnak) [inline packages/suite/src/services/metadata/DropboxProvider.ts]: This does not seem like something that should appear in production

File: packages/suite/src/services/metadata/DropboxProvider.ts [hunk 3]
```diff
@@ -18,6 +18,7 @@ class DropboxProvider implements AbstractMetadataProvider {
...
         const url = this.client.getAuthenticationUrl(getOauthReceiverUrl(), 'TODO:RandomToken');
```
Comment (prusnak) [inline packages/suite/src/services/metadata/DropboxProvider.ts]: `TODO:RandomToken` sounds likes something that should be fixed first

**Review by prusnak — CHANGES_REQUESTED** [review]

**Comment by prusnak** [comment]
Please also disable Google for now. It seems the new AppID needs to be "verified" first and we need more time to test this.

File: packages/suite/src/services/metadata/DropboxProvider.ts (hunk 3 above)
Comment (prusnak) [reply packages/suite/src/services/metadata/DropboxProvider.ts]: Resolved in 80b2a28

**Review by prusnak — APPROVED** [review]
LGTM

**Pull request merged**